cmake/Phylanx_SetupTarget.cmake

```diff
@@ -99,6 +104,21 @@ function(phylanx_setup_target target)
       string(REPLACE ";" ":" _prefix "${_prefix}")
     endif()
 
+    if(NOT target_STATIC_LINKING)
+      if(HPX_WITH_DYNAMIC_HPX_MAIN AND ("${_type}" STREQUAL "EXECUTABLE"))
+        if("${CMAKE_SYSTEM_NAME}" STREQUAL "Linux")
+          phylanx_info("phylanx_libs: " ${phylanx_libs})
+          set_target_properties(${target} PROPERTIES LINK_FLAGS "-Wl,-wrap=main")
```

Comment:
Do you really want to be setting these linker flags here? I know you're following HPX master pretty closely but this feels like unnecessarily using implementation details when we have alternatives (`hpx_setup_target`/`add_hpx_library`, and hopefully `HPX_LIBRARIES` back in action soon again) until we know that `target_link_libraries` works correctly.

Reply:
Those have no effect anyways (for whatever reason). Things now link but still fail at runtime (e.g. https://circleci.com/gh/STEllAR-GROUP/phylanx/32106).